enhancements/sig-policy/99-policy-placement-strategy/README.md

```diff
@@ -0,0 +1,269 @@
+    compliant (or after a configurable timeout):
+
+    | Status        | Description                                                                                                                                          |
+    | ------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- |
+    | `ToApply`     | No generation set                                                                                                                                    |
+    | `Progressing` | Current generation and `lastEvaluated` has not been updated                                                                                          |
+    | `Succeed`     | Compliant and current generation                                                                                                                     |
+    | `Failed`      | NonCompliant and current generation                                                                                                                  |
+    | `TimeOut`     | Current generation, `lastEvaluated` is less than `root-policy-timestamp`, </br>and time has passed beyond timeout specified in the `RolloutStrategy` |
+    | `Skip`        | (unused)                                                                                                                                             |
+
+  - (Stretch) Update the `Policy` CRD to contain a `RolloutStatus`. This will be used to provide an aggregated rollout
+    status for the root policy of `Progressing` (this would include `ToApply`), `Succeed`, or `Failed` (this would
+    include `TimeOut`).
+
+- **Snippet**
+
+  ```golang
+  type PlacementBinding struct {
+    metav1.TypeMeta   `json:",inline"`
+    metav1.ObjectMeta `json:"metadata,omitempty"`
+    // +kubebuilder:validation:Optional
+    BindingOverrides BindingOverrides `json:"bindingOverrides,omitempty"`
+    // +kubebuilder:validation:Optional
+    RolloutStrategy clusterv1alpha1.RolloutStrategy `json:"rolloutStrategy,omitempty"` // <-- New field
+    // This field provides the ability to select a subset of bound clusters
+    // +kubebuilder:validation:Optional
+    // +kubebuilder:validation:Enum=restricted
+    SubFilter SubFilter `json:"subFilter,omitempty"`
+    // +kubebuilder:validation:Required
+    PlacementRef PlacementSubject `json:"placementRef"`
+    // +kubebuilder:validation:Required
+    // +kubebuilder:validation:MinItems=1
+    Subjects []Subject              `json:"subjects"`
+    Status   PlacementBindingStatus `json:"status,omitempty"`
+  }
+  ```
+
+  ```golang
+  // ComplianceHistory defines compliance details history
+  type ComplianceHistory struct {
+    LastEvaluated metav1.Time `json:"lastEvaluated,omitempty" protobuf:"bytes,7,opt,name=lastEvaluated"`  // <-- New field
```

Comment:
Similar to my last comment, how is this field set?

Reply:
Moving thread to https://github.com/open-cluster-management-io/enhancements/pull/99#discussion_r1294997249